app/google-services.json

```diff
@@ -14,7 +14,7 @@
       },
       "oauth_client": [
         {
-          "client_id": "722367559564-orihetidnmp7btt5opql8nsd94mmqd6s.apps.googleusercontent.com",
+          "client_id": "722367559564-1u9jvql293afh70ns315uu1f7o1koinj.apps.googleusercontent.com",
```

Comment:
I had to change the app's `SHA1` encryption id to work with the google sign in, as the old one wasn't setup correctly in the firebase console.